plugins/utils/ccc.nix

```diff
@@ -0,0 +1,44 @@
+{
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "ccc";
+  originalName = "ccc.nvim";
+  defaultPackage = pkgs.vimPlugins.ccc-nvim;
+
+  maintainers = [helpers.maintainers.JanKremer];
+
+  settingsOptions = {
+    default_color = helpers.defaultNullOpts.mkString "#000000" ''
+      The default color used when a color cannot be picked. It must be HEX format.
+    '';
+
+    highlight_mode = helpers.defaultNullOpts.mkString "bg" ''
+      Option to highlight text foreground or background. It is used to output_line and highlighter.
+      Options: "fg" | "bg" | "foreground" | "background"
+    '';
+
+    lsp = helpers.defaultNullOpts.mkBool true ''
+      Whether to enable nvim-lsp support. The color information is updated in the background and the result is used by |:CccPick| and highlighter.
+    '';
+
+    highlighter.auto_enable = helpers.defaultNullOpts.mkBool false ''
+      Whether to enable automatically on BufEnter.
+    '';
+
+    highlighter.lsp = helpers.defaultNullOpts.mkBool true ''
+      If true, highlight using nvim-lsp. If LS with the color provider is not attached to a buffer, it falls back to highlight with pickers. See also |ccc-option-lsp|.
+    '';
```

Comment:
```suggestion
    highlighter = {
      auto_enable = helpers.defaultNullOpts.mkBool false ''
        Whether to enable automatically on BufEnter.
      '';
      lsp = helpers.defaultNullOpts.mkBool true ''
        If true, highlight using nvim-lsp. If LS with the color provider is not attached to a buffer, it falls back to highlight with pickers.
        See also |ccc-option-lsp|.
      '';
    };
```

Reply:
@traxys is `|ccc-option-lsp|` valid syntax? I've not seen it before in markdown.

Reply:
It's a vimdoc link I believe